blocks/CloudImageEditor/index.css

```diff
@@ -6,4 +6,5 @@ lr-cloud-image-editor {
   width: 100%;
   height: var(--modal-content-height-fill, 100%);
   overflow: hidden;
+  --cfg-cdn-cname: '';
```

Comment:
It's already declared for the solution here - https://github.com/uploadcare/uc-blocks/blob/chore/improve-workflows/solutions/cloud-image-editor/config.css#L3